pkgs/top-level/perl-packages.nix

```diff
@@ -22848,6 +22865,22 @@ let
     };
   };
 
+  XSParseSublike = buildPerlModule {
+    pname = "XS-Parse-Sublike";
+    version = "0.10";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/P/PE/PEVANS/XS-Parse-Sublike-0.10.tar.gz";
+      sha256 = "99a1bdda3ffa67514adb6aa189c902fa78dca41d778a42ae7079f604a045ac43";
+    };
+    perlPreHook = stdenv.lib.optionalString stdenv.isDarwin "export LD=$CC";
```

Comment:
Just a minor thing, but could you move the `perlPreHook` under `buildInputs`?

Reply:
Sure, while we're here, is there a document listing this correct ordering of these attrs?

Reply:
Thanks! Good question, I'm not sure there is one, but it's good to be consistent :-)